Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apk add --update ca-certificates
 COPY --from=builder /app/backup /usr/bin/backup
 
 COPY ./entrypoint.sh /root/
+COPY notifications.tmpl /etc/dockervolumebackup/notifications.d/00defaults.tmpl
```

Comment:
Sorry for forgetting about this so long, but instead of copying over the file here, we could also have Go embed it at compile time, which would have the advantage that we can guarantee that any template in `notifications.d` will always override the default values (right now someone could mount `00adjustments` and would be surprised to see no effect)

This would work a little like this:

```go
	var tmplErr error
	tmpl := template.New("")
    // add funcs ...
	tmpl, tmplErr = tmpl.Parse(defaultNotifications)
	if tmplErr != nil {
		return nil, fmt.Errorf("newScript: unable to parse default notifications templates: %w", err)
	}

	if _, err := os.Stat("/etc/dockervolumebackup/notifications.d"); err == nil {
		tmpl, tmplErr = tmpl.ParseGlob("/etc/dockervolumebackup/notifications.d/*.*")
		if tmplErr != nil {
			return nil, fmt.Errorf("newScript: unable to parse user defined notifications templates: %w", err)
		}
	}

	s.template = tmpl
```

Embedding the file would need to happen at top level and works like this:

```go
import _ "embed"

//go:embed notifications.tmpl
var defaultNotifications string
```

What do you think?

Reply:
I agree. That's neat that you can directly import plaintext files!